### PR TITLE
PSR-2 in production code

### PR DESCRIPTION
**Pull request by iwankgb**
Following types of changes have been applied to observe PSR-2:
* lines longer that 120 lines are broken into many lines (yes, it is a soft limit);
* method calls incorrectly broken into more then 1 line are fixed;
* some spacing issues are fixed too.

### CODE REVIEW

**Comment by taylorotwell** [comment]
I only tend to accept PSR-2 fixes that can be automated by php-cs-fixer as its more scalable going forward. Thanks

**Comment by GrahamCampbell** [comment]
Also, these changes aren't PSR-2. PSR-2 does NOT stipulate a hard line length.

File: src/Application.php [hunk 1]
```diff
@@ -370,7 +370,11 @@ protected function registerAuthBindings()
         });
 
         $this->singleton('auth.password', function () {
-            return $this->loadComponent('auth', 'Illuminate\Auth\Passwords\PasswordResetServiceProvider', 'auth.password');
+            return $this->loadComponent(
```
Comment (MacDada) [inline src/Application.php]: @GrahamCampbell You can argue that `PSR-2` does not require this…

File: src/Providers/CacheServiceProvider.php [hunk 2]
```diff
@@ -3,7 +3,8 @@
 use Illuminate\Cache\Console\ClearCommand;
 use Illuminate\Cache\CacheServiceProvider as BaseProvider;
 
-class CacheServiceProvider extends BaseProvider {
+class CacheServiceProvider extends BaseProvider
```
Comment (MacDada) [inline src/Providers/CacheServiceProvider.php]: @taylorotwell @GrahamCampbell …but this clearly is a violation.

**Pull request closed without merging**